microsoft/botframework-solutions
# [VA Template][TypeScript] Update deployment scripts

## Description
Update deployment scripts to follows TypeScript conventions.

## Testing Steps

1. Go to `./templates/Virtual-Assistant-Template/typescript/sample-assistant/package.json`
1. Open a powershell terminal.
1. Run ` .\deployment\scripts\deploy.ps1 -name "<BOT_NAME>" -resourceGroup "<RESOURCE_GROUP>" -location "<LOCATION>" -appId "<APP_ID>" -appPassword "<APP_PASSWD>" -luisAuthoringKey "<LUIS_KEY>" -luisAuthoringRegion "<LUIS_REGION>"`
1. Check that the deployment works.
